integration_tests/dbt_project.yml

```diff
@@ -55,9 +55,6 @@ seeds:
   jira_integration_tests:
     +column_types:
       _fivetran_synced: timestamp
-    board:
-      +column_types:
-        id: "{{ 'int64' if target.name == 'bigquery' else 'bigint' }}"
```

Comment:
This was removed as it was not used in the integration tests. 
![image](https://user-images.githubusercontent.com/74217849/191364681-c0dc9546-95ad-47dc-a53b-3c7d707c3289.png)